resolvers/README.md

```diff
@@ -42,7 +47,7 @@ exports.interfaceVersion = 3
 
 ### Required `resolve`
 
-Signature: `(source: string, file: string) => { found: boolean, path?: string | null }`
+Signature: `(source: string, file: string, options: { context: ChildContext | RuleContext, tsconfig?: TsConfigJsonResolved }) => { found: boolean, path?: string | null }`
```

Comment:
I don't like the idea of passing an object here. This creates multiple objects per resolve call and is not memory-friendly.